docs/components/BlockieAvatar.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 8
+---
+
+# BlockieAvatar
+
+Show a blockie (bar code profile icon) component for a given public address.
+
+If the address has an ENS avatar linked to it, this avatar will be displayed. Otherwise, a generated blockie will be shown.
+
+![BlockieAvatar Example](/img/BlockieAvatar.png)
```

Comment:
Should we just show the blockie only in image too, because people might get confused that it gives show address too, Maybe just : 

![Screenshot 2024-04-14 at 10 00 15 PM](https://github.com/scaffold-eth/se2-docs/assets/80153681/db38d4ae-b734-4237-b8bc-19fd78f8aa58)

Image when I used `<BlockieAvatar address="0x34aA3F359A9D614239015126635CE7732c18fDF3" size={24} />`  maybe we increase the size and take a pic ?